.github/workflows/push-to-upstream.yml

```diff
@@ -18,7 +18,6 @@ jobs:
       run: |
         eval "$(ssh-agent -s)"
         ssh-add - <<< "${{ secrets.GH_ACTION_SSH_KEY }}"
-        ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
```

Comment:
We can probably make this work by adding `mkdir -p ~/.ssh` before running this. I don't know whether this line is necessary or not.